lib/utils.js

```diff
@@ -156,12 +156,28 @@ const utils = {
       throw new Error(`No version-bump scope found for PR #${pr.number} (${pr.url})`)
     }
 
+    let scope
+
     if (matches.length > 1) {
-      throw new Error(`Too many version-bump scopes found for PR #${pr.number} (${pr.url})`)
+      // check for GFM checkboxes
+      const gfmCheckbockRegex = /-\s\[x\].*?#(\w+)#/gi
+
+      let selectedScopes = []
+      let checkboxMatches
+      while ((checkboxMatches = gfmCheckbockRegex.exec(pr.description)) !== null) {
```

Comment:
At first, I didn't really like the disparity between using `String.prototype.match` above and `RegExp.prototype.exec` here, but when I took a (very quick) stab at making them consistent, I decided it might not be worth the effort. So I'll leave it as-is.